shell/parse-url.sh

```diff
@@ -18,4 +18,7 @@ port=${port#:}
 user=${user_password%:*}
 password=${user_password#$user}
 password=${password#:}
-dbname=${rest}  # TODO do we need to strip any parameters from the rest of the URL?
+dbname=${rest%\?*}
+rest1=${rest#$dbname?}
+ssl=${rest1#ssl=} # TODO we only support ssl parameter from the rest of the URL
```

Comment:
This is fine for now, but spending a bit more effort to handle general query string parsing may be a wise move.  Well, even without doing any smart parsing, I think it may be more robust to pass the entire query string down to the JDBC url (say, via `$querystring`) and set the `PGSSLMODE=require` from db-parse if it contains `*ssl=true*`.  How does that sound?